### DISCUSSION TITLE
Test-DbaBuild and Update-DbaBuildReference - cannot download updated index

### DISCUSSION THREAD

**Discussion by andyl75** [dataplat/dbatools#9518] [Contributors]
Hi dbatools community,

I can open https://dataplat.github.io/assets/dbatools-buildref-index.json from a WebBrowser. 

However when I try to use Test-DbaBuild or Update-DbaBuildReference I get the following back.

$RTMBuilds = @('16.0.100');
Test-DbaBuild -Build $RTMBuilds -MaxBehind "0CU" -Update;

WARNING: [14:23:53][Get-DbaBuildReferenceIndexOnline] Couldn't download updated index from https://dataplat.github.io/assets/dbatools-buildref-index.json
WARNING: [14:23:53][Get-DbaBuildReferenceIndex] Index is stale, last update on: 2024-08-02T00:00:00, try the -Update parameter to fetch the most up to date index

Build          : 16.0.100
BuildLevel     : 16.0.100
BuildTarget    : 16.0.4135
Compliant      : False
CULevel        : 
CUTarget       : CU14
KBLevel        : 
MatchType      : Exact
MaxBehind      : 0CU
NameLevel      : 2022
SPLevel        : 
SPTarget       : RTM
SupportedUntil : 
Warning        : 

Update-DbaBuildReference
WARNING: [14:27:01][Get-DbaBuildReferenceIndexOnline] Couldn't download updated index from https://dataplat.github.io/assets/dbatools-buildref-index.json

**Comment by niphlod** [comment]
proxy ? 
for such cases you can download the file locally and then use

`Update-DbaBuildReference -LocalFile \\fileserver\Software\dbatools\dbatools-buildref-index.json`

**Reply by andyl75** [reply]
Worked. Thanks.
